### PR TITLE
Bug 1880784 - adding note about cafile

### PR DESCRIPTION
**Pull request by vwalek**
Please comment @kalexand-rh .

### CODE REVIEW

File: install/configuring_inventory_file.adoc [hunk 1]
```diff
@@ -1589,6 +1589,11 @@ openshift_master_named_certificates=[{"certfile": "/root/STAR.openshift.com.crt"
 openshift_master_overwrite_named_certificates=true
 ----
 
+[IMPORTANT]
+====
+The `cafile` certificate will be imported to `ca-bundle.crt` file on the masters during installation or during redeployment of certificates. The `ca-bundle.crt` is mounted to every pod running in OpenShift. Several OpenShift components will automatically trust the named certificates by default when accessing the `masterPublicURL`. Missing `cafile` in named certificates parameter will affect the functionality of Web Console and several other components.
```
Comment (kalexand-rh) [inline install/configuring_inventory_file.adoc:1594]:
```suggestion
The `cafile` certificate is imported to the `ca-bundle.crt` file on the masters during installation or during redeployment of certificates. The `ca-bundle.crt` file is mounted to every pod that runs in {product-title}. Several {product-title} components automatically trust the named certificates by default when they access the `masterPublicURL` endpoint. If you omit the `cafile` option from the certificates parameter, the functionality of Web Console and several other components is reduced.
```

@vwalek, maybe something like this? If there are more appropriate words to drop in, please do, but I think that the sentences should be structured more like this.

**Comment by vwalek** [comment]
@kalexand-rh ack and done ;) Thx

**Comment by kalexand-rh** [comment]
@gpei, will you please take a look?

File: install/configuring_inventory_file.adoc [hunk 2]
```diff
@@ -1589,6 +1589,11 @@ openshift_master_named_certificates=[{"certfile": "/root/STAR.openshift.com.crt"
 openshift_master_overwrite_named_certificates=true
 ----
 
+[IMPORTANT]
+====
+The `cafile` certificate is imported to the `ca-bundle.crt` file on the masters during installation or during redeployment of certificates. The `ca-bundle.crt` file is mounted to every pod that runs in {product-title}. Several {product-title} components automatically trust the named certificates by default when they access the `masterPublicURL` endpoint. If you omit the `cafile` option from the certificates parameter, the functionality of Web Console and several other components is reduced.
```
Comment (gpei) [inline install/configuring_inventory_file.adoc:1594]: There's no `cafile` parameter in above openshift_master_named_certificates option. We better to add a more clear example like:
openshift_master_named_certificates=[{"certfile": "/path/on/host/to/crt-file", "keyfile": "/path/on/host/to/key-file", "names": ["public-master-host.com"], "cafile": "/path/on/host/to/ca-file"}]

**Comment by gpei** [comment]
And we may also want to add this note in this section
https://docs.openshift.com/container-platform/3.11/install_config/certificate_customization.html#ansible-configuring-custom-certificates

In https://bugzilla.redhat.com/show_bug.cgi?id=1880784, we introduced a new option callled `openshift_named_certificate_omit_cafile `.
When `cafile` was not set in openshift_master_named_certificates, the playbook will quit directly.
By setting "openshift_named_certificate_omit_cafile=true" in addition, this check would be skipped.

**Comment by kalexand-rh** [comment]
@vwalek, if you update this PR, I'm happy to review it again.

File: install/configuring_inventory_file.adoc (hunk 2 above)
Comment (vwalek) [reply install/configuring_inventory_file.adoc:1594]: On it.

**Comment by vwalek** [comment]
> In https://bugzilla.redhat.com/show_bug.cgi?id=1880784, we introduced a new option callled `openshift_named_certificate_omit_cafile `.
> When `cafile` was not set in openshift_master_named_certificates, the playbook will quit directly.
> By setting "openshift_named_certificate_omit_cafile=true" in addition, this check would be skipped.

I don't want to add it in official docs as the variable is required. If cafile is not configured, the ansible check will report that and adds the variable to the message. The point is really not to skip the cafile.
I am afraid that if we will add the variable to the official docs, the cluster admins will add it intentionally to skip the check.
What do you think @kalexand-rh @gpei ?

**Comment by kalexand-rh** [comment]
> I don't want to add it in official docs as the variable is required. If cafile is not configured, the ansible check will report that and adds the variable to the message. The point is really not to skip the cafile.
> I am afraid that if we will add the variable to the official docs, the cluster admins will add it intentionally to skip the check.
> What do you think @kalexand-rh @gpei ?

@gpei, what do you think?

**Comment by gpei** [comment]
> I am afraid that if we will add the variable to the official docs, the cluster admins will add it intentionally to skip the check.
Yes, it makes sense to me

**Comment by kalexand-rh** [comment]
@gpei, do I have your approval to merge this change?

**Comment by gpei** [comment]
yeah, LGTM

**Pull request merged**